packages/foundry-sdk-generator/src/generate/betaClient/minifyBundle.ts

```diff
@@ -395,7 +565,24 @@ export class ProjectMinifier {
             continue;
           }
 
-          for (const namedExport of exportDeclaration.getNamedExports()) {
+          perNamedExport: for (
```

Comment:
Need to label this for loop for proper `continue`ing

Reply:
I did not know you could do this, how cool